portal/src/styles/variable.css

```diff
@@ -0,0 +1,16 @@
+:root {
+  --box-shadow-color: rgba(0, 0, 0, 0.1);
+  --widget-box-shadow: 0 0.5px 1px var(--box-shadow-color),
+    0 1.5px 3.5px var(--box-shadow-color);
+
+  --theme-primary-color: #176df3;
+  --theme-tertiary-color: #70a4f7;
+  --neutral-secondary-color: #605e5c;
+  --neutral-tertiary-color: #a19f9d;
+  --neutral-lighter-alt-color: #faf9f8;
+  --neutral-lighter-color: #f3f2f1;
+  --neutral-light: #edebe9;
+  --error-color: #e23d3d;
+  --white: #ffffff;
+  --black: #000000;
+}
```

Comment:
Our goal is to remove this file. Different variable requires different migration.

- `white` and `black`: Inline them
- `theme-primary-color` should be the primary blue color. Use `useTheme` to get access to it
- `theme-tertiary-color`: the name suggests that it is the tertiary shade of the primary color. Again, use `useTheme` to get access to it.
- `neutral-xxx-color`: Use `useTheme` to get access to them.
- `box-shadow-color`: Inline it
- `widget-box-shadow`: Inline it